taipy/gui/gui.py

```diff
@@ -2635,14 +2635,15 @@ def stop(self):
     def _get_autorization(self, client_id: t.Optional[str] = None, system: t.Optional[bool] = False):
         return contextlib.nullcontext()
 
-    def set_favicon(self, favicon_path: t.Union[str, Path]):
+    def set_favicon(self, favicon_path: t.Union[str, Path], state: t.Optional[State] = None):
         """Change the favicon for all clients.
 
         This function dynamically changes the favicon of Taipy GUI pages for all connected client.
         favicon_path can be an URL (relative or not) or a file path.
         The *favicon* parameter to `(Gui.)run()^` can also be used to change the favicon when the application starts.
```

Comment:
```suggestion
        TODO The *favicon_url* parameter to `(Gui.)run()^` can also be used to change the favicon when the application starts.
```
(doc is kind of incomplete - but will do in the future)

Reply:
the run parameter is named `favicon`